lib/features/sign_up/view/sign_up_view.dart

```diff
@@ -18,117 +19,161 @@ class SignUpScreen extends StatelessWidget {
+                      .withOpacity(0.3), // Adjust the opacity and color
+                ),
+              ),
+            ),
+            SafeArea(
+                child: Column(
+                    mainAxisAlignment: MainAxisAlignment.spaceBetween,
+                    children: [
+                  Expanded(
+                    child: Column(
+                        mainAxisAlignment: MainAxisAlignment.spaceBetween,
+                        children: [
+                          Row(
+                              mainAxisAlignment: MainAxisAlignment.end,
+                              children: [
+                                TextOnlyButton(
+                                  label: HatSpaceStrings.current.skip
+                                      .toUpperCase(),
+                                  onPressed: () {
+                                    context
+                                        .read<SignUpBloc>()
+                                        .add(const CloseSignUpScreen());
+                                  },
+                                  style: ButtonStyle(
+                                    foregroundColor:
+                                        MaterialStateProperty.all<Color>(
+                                      colorScheme.onPrimary,
+                                    ),
+                                  ),
                                 ),
-                                children: [
-                                  TextSpan(
-                                      style: textTheme.bodyMedium?.copyWith(
-                                          fontSize: 12,
-                                          color: HSColor.onSurface),
-                                      text: HatSpaceStrings.of(context)
-                                          .alreadyHaveAccount),
-                                  TextSpan(
-                                      style: const TextStyle(
-                                          fontWeight: FontWeight.w600,
-                                          decoration: TextDecoration.underline),
-                                      text: HatSpaceStrings.of(context).signIn,
-                                      recognizer: TapGestureRecognizer()
-                                        ..onTapDown = (details) {})
-                                ]))),
-                    TextOnlyButton(
-                      label: HatSpaceStrings.of(context).skip,
-                      onPressed: () {
-                        context
-                            .read<SignUpBloc>()
-                            .add(const CloseSignUpScreen());
-                      },
-                    ),
-                  ]),
-            )));
+                              ]),
+                          Padding(
+                            padding: const EdgeInsets.only(
+                                bottom: HsDimens.spacing80),
+                            child: SizedBox(
```

Comment:
Remove SizedBox here because SvgPicture widget already has the width and height properties